packages/relay/src/lib/eth.ts

```diff
@@ -517,6 +519,10 @@ export class EthImpl implements Eth {
           return predefined.INVALID_PARAMETER(0, `Invalid 'value' field in transaction param. Value must be greater than 0`);
         }
       } else {
+        // execute reverted: with no message comes from a contract deployment
+        if (e.stack.substring(0, 31) === EthImpl.errorContractReverted) {
```

Comment:
Q: @ebadiere do you still see errors with no messages once you rebased?
I'm wondering if the check for no message is still necessary.

Reply:
No, and this has been updated.